### ISSUE TITLE
Ender IO nonsense lag

### ISSUE THREAD

**Issue by SadRobo** [SleepyTrousers/EnderIO#5291]
#### Issue Description:

I'm creating a modpack and had some issues w/ Ender IO causing lag even when i'm w/ the game paused and in inventory. I'm running on 1.12.2 w/ the lastest version of all mods.
I tried to remove OreLib, UniDict, AE2 and JEI, but nothing changed.
I deleted and replaced every single mod.
Even when I run the modpack without Ender IO but w/ EnderCore it run perfectly.
Edit: I know this is caused by memory consume, so this is a Ender IO memory consuming problem(?)

#### What happens:

Game lagging when I load a world.
During the game, so as on inventory and "ESC" interface

#### What you expected to happen:

Play normally, w/o lag.

#### Steps to reproduce:

1.
2.
3.
...

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: 1.12.2-5.0.49
- EnderCore: 1.12.2-0.5.73
- Minecraft: 1.12.2
- Forge: 14.23.5.2847
- SpongeForge? no
- Optifine? yes
- Single Player and/or Server? Single Player

#### Your most recent log file where the issue was present: 

https://gist.github.com/SadRobo/f0c6d7f5f85c40681774f3889a482844

**Comment by VT-14** [comment]
You are 3 Ender IO versions behind.  It looks like the other stuff is actually up to date, including Forge.  Try updating.

If that doesn't help, profiling may help identify the issue.  [Sampler](https://www.curseforge.com/minecraft/mc-mods/sampler) or [vanilla's `/debug` command](https://minecraft.gamepedia.com/Commands/debug) (note; wiki is currently for 1.14 version) tend to be preferred methods.

"Lag" is not very descriptive.  Are you having FPS issues, TPS issues, or something else.  If FPS, are they constantly low, normally high with periodic freezes, etc?  

Other notes: It looks like you only have 2 GB allocated, which isn't a lot for a 1.12 pack.  Optifine is a general "please try without it."  Are you issues repeatable with only Ender IO (and dependencies) or do you need specific other mods to cause the issue?

**Comment by SadRobo** [comment]
I forgot to sat that I tested the last 3 versions and the same problem occurred.
Fps is normally high with periodic freeze (about each 2 second).
Ender IO w/o other mods works fine.
It's about 2,5GB allocated, w/o Ender IO everything works fine, with a smooth performance :/
> Modpack works fine w/o Ender IO
> Modpack have the sames issues even w/ Ender IO but w/o Optifine.
----- I wrote this like 2 hours ago and now...

I tried to put just 2 chunk and it was still lagging (unplayable at all)
BUT when I removed Ender IO and tested again, this was smoother, 70% was the max memory used I noticed. I changed to about 12 chunk and it was the same O.o
